core/client/fs/src/main/java/alluxio/client/file/options/OutStreamOptions.java

```diff
@@ -102,16 +116,21 @@ public OutStreamOptions(CreateFilePOptions options, AlluxioConfiguration alluxio
     }
   }
 
-  private OutStreamOptions(AlluxioConfiguration alluxioConf) {
+  private OutStreamOptions(ClientContext context, AlluxioConfiguration alluxioConf) {
     mCommonOptions = FileSystemOptions.commonDefaults(alluxioConf);
     mBlockSizeBytes = alluxioConf.getBytes(PropertyKey.USER_BLOCK_SIZE_BYTES_DEFAULT);
     mLocationPolicy = BlockLocationPolicy.Factory.create(
         alluxioConf.get(PropertyKey.USER_BLOCK_WRITE_LOCATION_POLICY),
         alluxioConf);
     mWriteTier = alluxioConf.getInt(PropertyKey.USER_FILE_WRITE_TIER_DEFAULT);
     mWriteType = alluxioConf.getEnum(PropertyKey.USER_FILE_WRITE_TYPE_DEFAULT, WriteType.class);
-    mOwner = SecurityUtils.getOwnerFromLoginModule(alluxioConf);
-    mGroup = SecurityUtils.getGroupFromLoginModule(alluxioConf);
+    try {
+      mOwner = context.getUserState().getUser().getName();
+      mGroup = CommonUtils.getPrimaryGroupName(mOwner, context.getConf());
+    } catch (IOException e) {
+      mOwner = "";
```

Comment:
(nit) Log an error message for the exception

Reply:
We could set the level to `DEBUG` if we ever need to trace security exceptions?